src/coreclr/jit/emitarm64.cpp

```diff
@@ -18525,6 +18812,14 @@ emitter::insExecutionCharacteristics emitter::getInsExecutionCharacteristics(ins
             }
             break;
 
+        // Not available in Arm Neoverse N2 Software Optimization Guide.
+        case IF_SVE_AG_3A: // ........xx...... ...gggnnnnnddddd -- SVE bitwise logical reduction (quadwords)
+        case IF_SVE_AJ_3A: // ........xx...... ...gggnnnnnddddd -- SVE integer add reduction (quadwords)
+        case IF_SVE_AL_3A: // ........xx...... ...gggnnnnnddddd -- SVE integer min/max reduction (quadwords)
+            result.insLatency    = PERFSCORE_LATENCY_20C;    // Invalid
```

Comment:
why is this "// Invalid"?

Reply:
These aren't in the manual. So, I wanted something to say it was just a randomly chosen high value. Maybe `// Placeholder` ?

Reply:
probably use `// TODO-SVE` that we use elsewhere too